samples/msal-node-samples/ElectronReactTestApp/README.md

```diff
@@ -0,0 +1,157 @@
+# MSAL Node Electron & React Sample
+
+An Electron and React application built with TypeScript that uses the MSAL Node library to acquire and store access tokens to authenticate with the Microsoft Graph API.
```

Comment:
We should have a shoutout here to electron-forge since the sample is built with it.

Reply:
Updated!